CRT.c

```diff
@@ -957,8 +960,22 @@ int CRT_scrollWheelVAmount = 10;
 
 ColorScheme CRT_colorScheme = COLORSCHEME_DEFAULT;
 
-ATTR_NORETURN
+
 static void CRT_handleSIGTERM(int sgn) {
+   CRT_terminateSignal = (sig_atomic_t)sgn;
+   CRT_terminateRequested = 1;
```

Comment:
This is only thread/async-safe if you guarantee write order with a memory barrier.

Does not handle re-entrance …